src/main/kotlin/org/radarbase/jersey/service/managementportal/MPProject.kt

```diff
@@ -0,0 +1,11 @@
+package org.radarbase.jersey.service.managementportal
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+data class MPProject(
+        @JsonProperty("projectName") val id: String,
+        @JsonProperty("humanReadableProjectName") val name: String? = null,
```

Comment:
Would be helpful to explicitly comments for these two properties explaining `project.id` is the project-name and `project.name` is the humanReadableName. I would more likely to be looking for `project.name` for project-name.